content/02.intro.md

```diff
@@ -5,7 +5,7 @@ Machine learning is a modern approach to problem-solving and task automation. In
 Artificial neural networks are a particular class of machine learning algorithms and models that evolved into what we now describe as "deep learning".
 Deep learning encompasses neural networks with many layers and the algorithms that make them perform well.
 These neural networks comprise artificial neurons arranged into layers and are modeled after the human brain, even though the building blocks and learning algorithms may differ [@doi:10.1038/s41583-020-0277-3].
-Each layer receives input from previous layers (the first of which represents the input data) and then transmits a transformed version of its input to the subsequent layer. 
+Each layer receives input from previous layers (the first of which represents the input data), and then transmits its own weighted output that then serves as input into subsequent layers of the network. 
```

Comment:
I used "transformed" here because I thought "weighted" might give the impression it was a linear scaling.  Is that potentially confusing?

Reply:
I agree that transformed is likely more accurate, but I actually do think weighted will be less confusing - for two reasons actually a) it is more intuitive to biologists who at least have some experience with linear "weights" and more importantly b) the word weights is used elsewhere throughout the manuscript to relate to this idea.

Reply:
The "weights" argument makes sense, but I think we should still go with technical correctness. Otherwise a careful reader can raise questions such as "if everything is just based on weighted sums, how can a neural network produce complex decisions?"

Maybe instead of 

>  then transmits its own weighted output

> then transmits a transformed version of its own weighted output

Reply:
I love this suggestion

> then transmits a transformed version of its own weighted output

seems like a really good way to get the best of both

I made the change